sdk/eventhub/event-hubs/perf.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+- name: operatingSystems
+  displayName: OperatingSystems (list of operating systems to run)
+  type: string
+  default: 'Linux'
+- name: PackageVersions
+  displayName: PackageVersions (regex of package versions to run)
+  type: string
+  default: '5|source'
+- name: Tests
+  displayName: Tests (regex of tests to run)
+  type: string
+  default: '^(subscribe|send)$'
+- name: Arguments
+  displayName: Arguments (regex of arguments to run)
+  type: string
+  default: '(1024)'
+- name: Iterations
+  displayName: Iterations (times to run each test)
+  type: number
+  default: '5'
+- name: AdditionalArguments
+  displayName: AdditionalArguments (passed to PerfAutomation)
+  type: string
+  default: ' '
+
+extends:
+  template: /eng/pipelines/templates/jobs/perf.yml
+  parameters:
+    ServiceDirectory: eventhub
```

Comment:
```suggestion
    ServiceDirectory: eventhub/event-hubs
```

Reply:
Interesting... the pipeline was working fine with just "eventhub"